🛠️ Refactor Sentieon Joint Call for Beta Prod (#1)
kids-first/Kids-First-Sentieon-Joint-Cohort-Genotyping-Workflow: docs/KF_NOTEBOOK_JG_WORKFLOW.md

# Kids First-Sentieon Joint Cohort Call Notebook

> Due to platform limitations, running the Preferred workflow may not be feasible when cohorts exceed 2200 samples, due to scatter job size limitations.

Reviewer: Idk if this has an obvious answer, but what should users do if the cohort is > 2200? Should they break up the samples into smaller cohorts and do multiple runs? Or is the breaking up of chromosomes for the entire cohort the solution? It's unclear when reading this section

Author: Oh, well, it's to run this one up to a certain point, probably like 3500. Lemme double check, I put some hints elsewhere, but you get to a point where, for this platform, you'd have to either use smaller cohorts (but sometimes that's just not feasible as perhaps all samples have to be run together and can't be further sub-categorized), or focus on certain regions, which this workflow does not do. That might be a future update

# Kids First-Sentieon Joint Cohort Call Notebook

> Due to platform limitations, running the easier workflow may not be feasible when cohorts exceed 200 samples, due to scatter job size limitations.

Reviewer 2: 200 samples? Not 2000?

> [This notebook](../workflow/kf-joint-cohort-call-advanced-setup.ipynb) can be used to set up the split by chromosome jobs, in which N jobs are created to split X files per instance, depending on yor math.

Reviewer 2: ```suggestion
[This notebook](../workflow/kf-joint-cohort-call-advanced-setup.ipynb) can be used to set up the split by chromosome jobs, in which N jobs are created to split X files per instance, depending on your math.
```

> Then with sensible tagging and sorting, the outputs from these jobs can be reliable pass to the Sentieon GVCFtyper to create the final product.

Reviewer 2: ```suggestion
Then with sensible tagging and sorting, the outputs from these jobs can be reliably passed to the Sentieon GVCFtyper to create the final product.
```

> In the end, you'll want to delete all files with the `INTERMEDIATE` tag to avoid excessive storage costs on files that are likely cheaper to regenerate as needed than store indefinitely

Reviewer 2: ```suggestion
In the end, you'll want to delete all files with the `INTERMEDIATE` tag to avoid excessive storage costs. These files can be quickly and cheaply regenerated, as needed.
```
